app/assets/javascripts/oxalis/api/api_latest.js

```diff
@@ -566,8 +567,13 @@ class DataApi {
       throw new Error(messages["mapping.too_few_textures"]);
     }
 
-    const layer = this.__getLayer(layerName);
-    layer.cube.setMapping(mapping);
+    // TODO: As there is at most one segmentation layer now, the layerName is unneccessary
```

Comment:
Hm, I think, we are talking about different things. I only wanted to say that I would keep the `layerName` parameter in the `setMapping` signature for now, as this seems more future-proof. I'm referring to your code comment which says that this parameter could be removed. But then, we need to change the API version and in case we support multiple segmentations at some point, we need to re-change it.

So, the actual changes made in this file are fine, but I'm saying that I don't think that the "todo" comment is necessary.